src/test/java/bio/terra/service/filedata/google/firestore/FireStoreFileDaoTest.java

```diff
@@ -55,14 +54,14 @@ class FireStoreFileDaoTest {
   @Autowired private FireStoreUtils fireStoreUtils;
   @Autowired private ConnectedOperations connectedOperations;
   @Autowired private ConnectedTestConfiguration testConfig;
-  @MockBean private IamProviderInterface samService;
+  @MockitoBean private IamProviderInterface samService;
   private String datasetId;
   private Firestore firestore;
   private List<FireStoreDirectoryEntry> directoryEntries;
   private List<FireStoreFile> files;
   private List<String> objects;
 
-  @BeforeAll
+  @BeforeEach
```

Comment:
I'm not sure if this one is strictly necessary, but it was done in the other PR.